ggml.c

```diff
@@ -8544,15 +8620,20 @@ static void ggml_compute_forward_dup_f32(
                 ggml_from_float_t const quantize_row_q = type_traits[dst->type].from_float;
 
                 size_t id = 0;
-                size_t rs = nb0 * (ne00 / ggml_blck_size(dst->type));
+                const size_t rs = nb0 * ne00_padded / blck_size;
                 char * dst_ptr = (char *) dst->data;
+                float src0_padded[ne00_padded];
```

Comment:
I don't understand why the CI runs are failing. From what I can tell the issue is that for some reason it tries to allocate an array of size 0 here. This would imply `ne00_padded == 0` which in turn would imply `ne00 == 0`. Is there a situation in which calling this function with that configuration actually makes sense?

Reply:
MSVC doesn't support VLA, the size of the array must be a compile-time constant.

Reply:
Maybe Install the clang extension in Visual Studio, and use it instead of MSVC.